docs/RFCS/range_leases.md

```diff
@@ -101,6 +114,23 @@ node ID and epoch match the last committed lease, the command will be
 applied; otherwise it will be rejected.
 
 
+# Performance implications
+
+We expect traffic proportional to the number of nodes in the system.
+With 1,000 nodes and a 9s liveness duration threshold, we expect every
+node to do a conditional put to update the heartbeat timestamp every
+7.2s. That would correspond to ~140 reqs/second, a not-unreasonable
+load for this function.
```

Comment:
Elsewhere in the document we talk about a 10,000 range table requiring 1,388 Raft commits per second. It would be nice to reiterate how many Raft ops/sec the old (non-epoch-based range leases) would require for a 1,000 node cluster with 10,000 replicas per node (3.3 million ranges).